sim/side.ts

```diff
@@ -629,19 +629,25 @@ export class Side {
 
 			switch (choiceType) {
 			case 'move':
-				let targetLoc = 0;
+				const original = data;
+				const error = () => this.emitChoiceError(`Conflicting arguments for "move": ${original}`);
+				let targetLoc: number | undefined;
 				let megaOrZ = '';
 				while (true) {
 					if (/\s-?[1-3]$/.test(data)) {
```

Comment:
```suggestion
					if (/\s(?:-|\+)?[1-3]$/.test(data)) {
```

Reply:
No reason not to support it as an option. Will be useful if we get another move ending in a number.

Reply:
Done.